### PR TITLE
[TerritorialCouncil][Referent] Add PoliticalCommittee info and filters

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: front/style/pages/_territorial-council.scss [hunk 1]
```diff
@@ -1,6 +1,16 @@
 .territorial-council {
     padding-bottom: 100px;
 
+    &__label {
+        &--elected {
```
Comment (ottaviano) [inline front/style/pages/_territorial-council.scss:5]: could you try to use the class `elected-tag` instead?

File: src/Repository/TerritorialCouncil/TerritorialCouncilMembershipRepository.php [hunk 2]
```diff
@@ -202,10 +202,30 @@ private function createFilterQueryBuilder(MembersListFilter $filter): QueryBuild
...
+                $qb
+                    ->leftJoin('adherent.politicalCommitteeMembership', 'pcm')
+                    ->leftJoin('pcm.qualities', 'pcQuality')
+                    ->andWhere('(quality.name in (:qualities) OR pcQuality.name IN (:pcQualities))')
+                    ->setParameter('qualities', $tcQualities)
+                    ->setParameter('pcQualities', $pcQualities)
+                ;
+            } else {
+                $qb
+                    ->andWhere('quality.name in (:qualities)')
```
[the remark this answers is not in the document]
Comment (illusionOfParadise) [reply src/Repository/TerritorialCouncil/TerritorialCouncilMembershipRepository.php:225]: Yes, we check it always, but in the case of PC qualities we need to apply `OR` condition: `OR` PC, `OR` TC qualities.

[the remark this answers is not in the document]
Comment (ottaviano) [reply src/Repository/TerritorialCouncil/TerritorialCouncilMembershipRepository.php:225]: oh yes, my bad 👍

**Comment by illusionOfParadise** [comment]
[ch12496]

**Review by ottaviano — APPROVED** [review]

**Review by kevin-verschaeve — APPROVED** [review]

**Pull request merged**